disnake/errors.py

```diff
@@ -185,6 +189,32 @@ class DiscordServerError(HTTPException):
     pass
 
 
+class RatelimitTooLong(DiscordException):
+    """Exception that's raised when waiting for a ratelimit would take longer than the
+    configured maximum time (see :attr:`Client.max_ratelimit_wait <Client>`).
+
+    Attributes
+    ----------
+    reset_after: float
```

Comment:
What's the purpose of this when `reset_at` exists? This value would only be valid at the time the exception is raised